cgl.h

```diff
@@ -5829,6 +5840,7 @@ void CGL_framebuffer_destroy(CGL_framebuffer* framebuffer)
     free(framebuffer);
 }
 
+#ifndef CGL_EXCLUDE_WINDOW_API
```

Comment:
Thing thing here rather than excluding the entire CGL_framebuffer_bind when WINDOW API is not being used is it not better to just do something like:
```
void CGL_framebuffer_bind(CGL_framebuffer* framebuffer)
{
    if(framebuffer->is_default)
    {
        CGL_int width = 512, height = 512;// some default values 
#ifdef CGL_WASM
        // TODO
#else

#ifndef CGL_EXCLUDE_WINDOW_API
        CGL_window_get_size((CGL_window*)framebuffer->user_data, &width, &height);
#endif
#endif
        glViewport(0, 0, width, height);
        glBindFramebuffer(GL_FRAMEBUFFER, 0);
    }
    else
    {
        glBindFramebuffer(GL_FRAMEBUFFER, framebuffer->handle); 
        glViewport(0, 0, framebuffer->width, framebuffer->height);
        if(framebuffer->color_attachment_count > 0)
        {
            GLenum buffers[8] = { GL_COLOR_ATTACHMENT0, GL_COLOR_ATTACHMENT1, GL_COLOR_ATTACHMENT2, GL_COLOR_ATTACHMENT3, GL_COLOR_ATTACHMENT4, GL_COLOR_ATTACHMENT5, GL_COLOR_ATTACHMENT6, GL_COLOR_ATTACHMENT7 };
        	glDrawBuffers(framebuffer->color_attachment_count, buffers);
        }            
    }
}
```

In this case the API will still work even if the user is not using CGL_WINDOW_API